### DISCUSSION TITLE
Custom page with custom layout & records from multiple models

### DISCUSSION THREAD

**Discussion by wpas81** [filamentphp/filament#9573] [Help]
Hi

I'm reworking a custom made backend into FilamentPHP (wonderful piece of coding by the way!)

For all regular Resource pages this is nog issue but I've got one 'sales overview'-page which lists a bunch of data from multiple models with a small form in the top to filter the results.

This is not a table-page more an InfoList but not limited to a single Model (and many results rely on relationship-data and must be filtered, flattened, etc...).

Is this possible? How can I achieve this? Preferrably with my own custom view (although an InfoList page would do)

thanks in advance

**Comment by leandrocfe** [comment]
You can create [custom pages](https://filamentphp.com/docs/3.x/panels/resources/custom-pages#overview). Pages are livewire components.

**Comment by wpas81** [comment]
yes I was  looking into that but from the documentation it doesn't seem I can also create a custom layout and that a custom page behaves exactly like a resource-page: you'll define a table-view, form etc

I need a few datasets eg $data1 = Model1::where([...])->get(), $data2 = Model2::(....), ... and a custom layout to display this (I'll do a bunch of calculations/filters on the collections to display various metrics

**Comment by wpas81** [comment]
I mananaged with a custom widget. Was hoping for a more elegant solution but this will do
